src/main.ts

```diff
@@ -142,7 +142,11 @@ async function run() {
             await exec("git", ["show"], inDownstreamOptions);
         }
     } catch (error) {
-        core.setFailed(error.message);
+        if (error instanceof Error) {
```

Comment:
This change works around compilation failure on `Version 4.4.3` of TypeScript where error is typed `unknown`.